tests/infra_test.go

```diff
@@ -682,6 +693,8 @@ var _ = Describe("[Serial]Infrastructure", func() {
 			}
 
 			errorsChan := make(chan error)
+			By("Scraping the Prometheus endpoint")
+			metricsURL := fmt.Sprintf("https://%s/metrics", net.JoinHostPort(ip, strconv.Itoa(8443)))
```

Comment:
Please extract to a method. I duplicate [this](https://github.com/kubevirt/kubevirt/pull/4307/commits/952eb7f1d9b59c9253de87f5068aedf4a9d24844#diff-c95f11746affb831bc43f837f8be65ad753336c090e3b181531ec00bf7636d7dR452).